Migration to NodeJS (#21)
leibatt/scalable-vega: README.md

## Installation
1. Install and start PostgreSQL.

> 2. Create a PostgreSQL database named `scalable_vega`, e.g., `createdb scalable_vega`
>     1. You don't need to install or setup for DuckDB

Reviewer: ```suggestion
2. Create a PostgreSQL database named `scalable_vega`, e.g., `createdb scalable_vega`. You don't need to do anything if you use DuckDB.
```

3. Run `cd /path/to/dev/repos`.
4. Run `git clone git@github.com:leibatt/scalable-vega.git`.

> 5. Run `yarn` to javacript install dependencies.

Reviewer: ```suggestion
5. Run `yarn` to install NPM dependencies.
```

6. For using prepopulated database, look at additional notes.

## Running Application Server
1. Run `node server.js` 

>     1. Default database used is DuckDB, if you want to use postgres as your database run `node server.js pg`

Reviewer: ```suggestion
    1. The default database used is DuckDB. If you want to use postgres as your database run `node server.js pg`.
```
